tool/tctl/main.go

```diff
@@ -31,6 +31,7 @@ func main() {
 		&common.TopCommand{},
 		&common.AccessRequestCommand{},
 		&common.AppsCommand{},
+		&common.DBCommand{},
```

Comment:
Don't forget to add this to Enterprise `tctl` as well since it's a separate binary.

Reply:
Yeah, I have it in a branch, will add to this PR.

Reply:
https://github.com/gravitational/teleport.e/pull/199